src/abstractdataframe/iteration.jl

```diff
@@ -411,6 +411,8 @@ julia> df
 """
 function mapcols!(f::Union{Function,Type}, df::DataFrame)
     # note: `f` must return a consistent length
+    (ncol(df) == 0) && return df # skip if no columns
```

Comment:
```suggestion
    ncol(df) == 0 && return df # skip if no columns
```

Reply:
Just for me to know for the future: is it a recommended codestyle for dataframes?
I'm fine with it,  just with all these &, &&, == precedence subtleties I prefer to be on the safe side in my scripts.

Reply:
You probably saw that I left the same comment earlier, but I am usually more flexible than @nalimilan.
The general coding style we try to follow is to use what Julia Base recommends (and in this case this is the recommended style, though with `&` indeed parentheses would be needed)